tests/fs/api.private.node.test.ts

```diff
@@ -1,9 +1,14 @@
 import expect from "expect"
 import * as fc from "fast-check"
 
+// @ts-ignore
+import mockery from "mockery"
```

Comment:
I can't find a usage of this. Can it be deleted?

Reply:
Oh whoops, yeah was part of the "proxying" experiments.